Hackney.Shared.HousingSearch/Domain/Tenure/TempAccommodationInfo.cs

```diff
@@ -17,6 +18,7 @@ public TempAccommodationInfo()
         private TempAccommodationInfo(QueryableTempAccommodationInfo TempAccommodationInfo)
         {
             BookingStatus = TempAccommodationInfo?.BookingStatus;
+            AssignedOfficer = TempAccommodationInfo?.AssignedOfficer;
```

Comment:
This is fine now, but that's because the queryable model that is used as input to this ctor contains a non-queryable (domain) model inside it.

When the `QueryableTempAccommodationInfo` model is changed to contain queryable assigned officer _(rather than domain assigned officer)_ this will line will need to be amended to account for conversion.

Reply:
Addressed in [cebad9e](https://github.com/LBHackney-IT/housing-search-shared/pull/74/commits/cebad9ecced7120c18e3bbcd9cd2d26a571dc601)